src/linagora.esn.unifiedinbox/app/components/inbox-configuration/filter-definition/configuration-filter-definition.less

```diff
@@ -18,6 +18,10 @@
     padding: 0;
   }
 
+  recipients-auto-complete .tags .tag-item {
+    max-width: none;
```

Comment:
I'm just curious why do we need this?

Because `none` is the default for this attribute https://www.w3schools.com/cssref/pr_dim_max-width.asp

Reply:
because it is defined in other places